package org.forestwizard.cities.exception;

public class ResourceLoaderException extends Exception {
    public ResourceLoaderException(String msg) {
        super(msg);
    }

    public ResourceLoaderException(String msg, Throwable cause) {
        super(msg, cause);
    }
}
